### PR TITLE
Scylla bench: fixes and regression analysis support

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2045,14 +2045,24 @@ def _parse_bench_summary(lines):
...
+            if line.startswith('Latency:') or ':' not in line:
                 continue
             if not enable_parse:
                 continue
 
             split_idx = line.index(':')
             key = line[:split_idx].strip()
             value = line[split_idx + 1:].split()[0]
+            try:
+                value = value.decode('utf-8').encode('utf-8').replace('\xc2\xb5s', 'mcs')
```
Comment (bentsi) [inline sdcm/cluster.py]: This looks ugly, sorry :)
What is the reason for that?

File: sdcm/cluster.py [hunk 2]
```diff
@@ -2258,23 +2270,20 @@ def node_run_stress_bench(node, loader_idx, stress_cmd, node_list):
         return queue
 
     def kill_stress_thread_bench(self):
-        kill_script_contents = 'PIDS=$(pgrep -f cassandra-stress) && pkill -TERM -P $PIDS'
-        kill_script = script.TemporaryScript(name='kill_cassandra_stress.sh',
+        kill_script_contents = 'PIDS=$(pgrep -f scylla-bench) && pkill -TERM -P $PIDS'
```
Comment (bentsi) [inline sdcm/cluster.py]: why not `pkill -f  -TERM scylla-bench`?

File: sdcm/cluster.py (hunk 1 above)
Comment (larisau) [reply sdcm/cluster.py]: agree, I'll replace it by something more readable

**Comment by roydahan** [comment]
@larisau please add the longevity yaml you used to test it to the PR.

**Comment by larisau** [comment]
I'll send yaml file later.

**Pull request merged**